Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: build test push push-image
+
+IMAGE_NAME ?= docsonnet
+IMAGE_PREFIX ?= jsonnet-libs
```

Comment:
do we have this docker hub account?

Reply:
Not yet afaik, didn't enable pushing yet either.